Adyen.EcommLibrary/Service/Payment.cs

```diff
@@ -33,13 +34,30 @@ public PaymentResult Authorise(PaymentRequest paymentRequest)
             return paymentResult;
         }
 
+        public async Task<PaymentResult> AuthoriseAsync(PaymentRequest paymentRequest)
+        {
+            PaymentResult paymentResult = null;

```

Comment:
```suggestion
            
                var jsonRequest = Util.JsonOperation.SerializeRequest(paymentRequest);
                var jsonResponse = await _authorise.RequestAsync(jsonRequest);
                return JsonConvert.DeserializeObject<PaymentResult>(jsonResponse);
```

Reply:
If we remove this try catch we need to remove it for Authorise too. I would suggest 
try { ... }catch(Exception ex) { throw ex; } similar to Authorise and Authorise3D

Reply:
If we remove this try catch we need to remove it for Authorise too. I would suggest 
try { ... }catch(Exception ex) { throw ex; } similar to Authorise and Authorise3D

Reply:
If we remove this try catch we need to remove it for Authorise too. I would suggest 
try { ... }catch(Exception ex) { throw ex; } similar to Authorise and Authorise3D

Reply:
If we remove this try catch we need to remove it for Authorise too. I would suggest 
try { ... }catch(Exception ex) { throw ex; } similar to Authorise and Authorise3D

Reply:
If we remove this try catch we need to remove it for Authorise too. I would suggest 
try { ... }catch(Exception ex) { throw ex; } similar to Authorise and Authorise3D

Reply:
If we remove this try catch we need to remove it for Authorise too. I would suggest 
try { ... }catch(Exception ex) { throw ex; } similar to Authorise and Authorise3D